src/styles/_catalog.scss

```
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition-duration: 300ms;
```

Рекомендую бiльш точно вказувати, до яких саме властивостей ти збираєшся застосовувати transition.
```suggestion
    transition-property: ;
    transition-duration: 300ms;
```